### PR TITLE
CLI: initial save/read paired dataset support for `melodies-monet run`

### PR DESCRIPTION
**Pull request by zmoon**
1. if `analysis.read` is present in the config, read saved paired datasets instead of pairing
2. if `analysis.save` is present in the config, save paired datasets

This is something I've been using for a while, and it's been working fine.

### CODE REVIEW

File: melodies_monet/_cli.py [hunk 1]
```diff
@@ -146,7 +146,14 @@ def run(
         an.open_obs()
 
     with _timer("Pairing"):
```
Comment (zmoon) [inline melodies_monet/_cli.py:148]: Maybe should have separate message for each case here

Comment (zmoon) [reply melodies_monet/_cli.py:148]: Oh but I guess the file names are already printed

**Comment by zmoon** [comment]
linkcheck failed for <https://www3.epa.gov/ttn/naaqs/standards/ozone/data/20140131healthrea4a.pdf>, which is indeed currently unreachable

**Comment by bbakernoaa** [comment]
@rschwant @colin-harkins can one of you take a look at this

**Review by colin-harkins — APPROVED** [review]
This looks good to me. Thanks for making a test for this.

**Pull request merged**